src/components/cc-kv-hash-explorer/cc-kv-hash-explorer.js

```diff
@@ -0,0 +1,579 @@
+        ?skeleton=${skeleton}
+        ?disabled=${this.disabled}
+        @cc-button:click=${this._onCopyKeyButtonClick}
+        >${i18n('cc-kv-hash-explorer.element.copy')}</cc-button
+      >
+    `;
+  }
+
+  _renderAddForm() {
+    const loading = this.state.type === 'loading';
+    const adding = this.state.type !== 'loading' && this.state.addForm.type === 'adding';
+    const readonly = loading || adding;
+
+    return html`
+      <form class="add-form" ${ref(this._addFormRef)} ${formSubmit(this._onAddFormSubmit)}>
+        <cc-input-text
+          name="field"
+          label=${i18n('cc-kv-hash-explorer.add-form.element-field')}
+          ?readonly=${readonly}
+          ?disabled=${this.disabled}
+          inline
+        ></cc-input-text>
+        <cc-input-text
+          name="value"
+          label=${i18n('cc-kv-hash-explorer.add-form.element-value')}
+          ?readonly=${readonly}
+          ?disabled=${this.disabled}
+          inline
+          multi
+        ></cc-input-text>
+        <cc-button
+          type="submit"
+          a11y-name=${i18n('cc-kv-hash-explorer.add-form.submit.a11y')}
+          .icon=${iconAdd}
+          ?waiting=${adding}
+          ?disabled=${loading || this.disabled}
+          >${i18n('cc-kv-hash-explorer.add-form.submit')}</cc-button
+        >
+      </form>
+    `;
+  }
+
+  /**
+   * @return {Array<{state: CcKvHashElementState, skeleton: boolean}>}
+   */
+  _getElements() {
+    switch (this.state.type) {
+      case 'loading':
+        return SKELETON_ELEMENTS;
+      case 'loaded':
+        return this.state.elements.map((state) => ({ state, skeleton: false }));
+      case 'loading-more':
+        return [...this.state.elements.map((state) => ({ state, skeleton: false })), ...SKELETON_ELEMENTS];
+    }
+  }
+
+  static get styles() {
+    return [
+      skeletonStyles,
+      // language=CSS
+      css`
+        :host {
+          display: block;
+        }
+
+        .wrapper {
+          display: flex;
+          flex-direction: column;
+          height: 100%;
+        }
+
+        .top-bar {
+          align-items: center;
+          border-bottom: 1px solid var(--cc-color-border-neutral-strong);
+          display: grid;
+          gap: 0.5em;
+          grid-auto-columns: 1fr auto;
+          grid-auto-flow: column;
+          padding: 0.5em;
+        }
+
+        .elements {
+          flex: 1;
+        }
+
+        .element,
+        .header {
+          align-items: center;
+          display: grid;
+          gap: 0.5em;
+          grid-auto-columns: 1fr 1fr auto auto auto;
+          grid-auto-flow: column;
+          padding: 0.25em 0.5em;
+          width: 100%;
+        }
+
+        .element-value {
+          align-items: center;
+          display: grid;
+          gap: 0.5em;
+          grid-template-columns: 1fr auto;
+        }
+
+        .element-field,
+        .element-value-span {
+          overflow: hidden;
+          text-overflow: ellipsis;
```

Comment:
If I could easily add a copy-to-clipboard button here, I would have done it and it could have been the right solution.

I will create an issue for that.